### PR TITLE
Tei id extractor config and terraform

### PR DESCRIPTION
**Pull request by alicefuzier**
Depends on #1686 
Part of https://github.com/wellcomecollection/catalogue-pipeline/issues/1536

### CODE REVIEW

File: tei_adapter/tei_id_extractor/src/main/resources/application.conf [hunk 1]
```diff
@@ -1,3 +1,14 @@
 aws.sqs.queue.url=${?queue_url}
 aws.sns.topic.arn=${?topic_arn}
 tei.github.token=${?github_token}
+aws.rds.host=${?db_host}
+aws.rds.password=${?db_password}
+aws.rds.port=${?db_port}
+aws.rds.username=${?db_username}
+aws.rds.maxConnections=${?max_connections}
+tei.id_extractor.bucket=${?bucket}
+tei.id_extractor.database=pathid
```
Comment (kenoir) [inline tei_adapter/tei_id_extractor/src/main/resources/application.conf:10]: Why not also parameterize these?

File: tei_adapter/terraform/provider.tf [hunk 2]
```diff
@@ -2,6 +2,6 @@ provider "aws" {
   region = local.aws_region
 
   assume_role {
-    role_arn = "arn:aws:iam::760097843905:role/platform-developer"
+    role_arn = "arn:aws:iam::760097843905:role/platform-admin"
```
Comment (kenoir) [inline tei_adapter/terraform/provider.tf:5]: Why must this be admin?

**Review by kenoir — APPROVED** [review]

File: tei_adapter/terraform/provider.tf (hunk 2 above)
Comment (alicefuzier) [reply tei_adapter/terraform/provider.tf:5]: argh thanks. it's a leftover when I tried to destroy the stack and recreate it. developer can't do it. Will change

File: tei_adapter/tei_id_extractor/src/main/resources/application.conf (hunk 1 above)
Comment (alicefuzier) [reply tei_adapter/tei_id_extractor/src/main/resources/application.conf:10]: no reason really. They're not parametrised in the id minter and I just copied that I think. Will change

**Pull request merged**